lineman/app/modules/group_page/invitations_modal_controller.coffee

```diff
@@ -38,6 +38,7 @@ angular.module('loomioApp').controller 'InvitationsModalController', ($scope, $m
     $scope.isDisabled = false
     $scope.invitations = []
     $modalInstance.close()
+    Records.memberships.fetchByGroup $scope.group
```

Comment:
please add comment:
reget memberships for group, as this are not returned with invitations create response